### DISCUSSION TITLE
Why did Termux not use `Ctrl+Shift+C/V` to copy/paste? Why did we adopt `Ctrl+Alt+C/V`?

### DISCUSSION THREAD

**Discussion by knyipab** [termux/termux-app#3928] [General]
Is there any reasons to design Termux to use `Ctrl+Alt+C/V` but not `Ctrl+Shift+C/V`?

As I get so used to Linux terminal, I always press the wrong shortcuts when switch to Termux. I wonder why Termux is not consistent with the mainstream Linux terminal shortcuts. Or is there configuration option to modify that?

When use Termux:X11, I never encounter problem using `Ctrl+Shift+C/V` in GUI terminal, so I don't quite see the point to not allow the Linux convention.

**Comment by Coderx7** [comment]
I guess its because the shift key is not present in the auxiliary key sections, so they probably said, we have ctrl, alt, so lets use that instead! 
personally I, like you, prefer the normal crtl+shift+c/v!  its just natural to use try to do that!

**Reply by knyipab** [reply]
By "auxiliary key sections", do you mean the extra keys bar below? Lol, that doesn't sound to be a good reason why not `Ctrl+Shift` for hardware keyboard (and DeX) user.

**Reply by Coderx7** [reply]
yup, that's it. this is what occurred to me, as I couldn't find any other plausible reasons that would necessitate such weird choice imho!

**Reply by luisdavim** [reply]
I think it makes sense not always I'm using a physical keyboard...

**Reply by RohitKushvaha01** [reply]
> yup, that's it. this is what occurred to me, as I couldn't find any other plausible reasons that would necessitate such weird choice imho!

I think it's a android limitation apps doesn't receive shift key press on Android

**Reply by knyipab** [reply]
I suppose shift key is not the limitation cuz Termux:X11 does receive that from hardware keyboard.
